orga/app/services/url.js

```diff
@@ -62,4 +74,14 @@ export default class Url extends Service {
     }
     return url;
   }
+
+  _computeShowcaseWebsiteUrl({ en: englishPath, fr: frenchPath }) {
+    const currentLanguage = this.intl.t('current-lang');
+
+    if (this.isFrenchDomainExtension) return `${PIX_FR_DOMAIN}${frenchPath}`;
```

Comment:
Remarque : une méthode privée qui dépend d'une méthode de plus haut niveau `this.isFrenchDomainExtension`,  qui elle même dépend d'un service, est-ce que ça risque pas de poser des soucis ? Alors qu'on veut juste renvoyer la bonne url suivant l'extension

Reply:
Hmm, c'est vrai qu'il y a une dépendance avec le service mais comme on doit récupérer `currentDomain` pour tester, je ne vois pas comment faire autrement 🤔 

Je peux passer currentDomain en paramètre de la fonction _computeShowcaseWebsiteUrl pour enlever le `this` et faire un test `if(currentDomain === FRENCH_DOMAIN_EXTENSION)`. T'en penses quoi ?

Reply:
ça pourrait être cool, comme ça on aurait moins d'intelligence dans cette fonction privée (faudrait que je regarde un peu de doc à ce sujet, je suis pas sure qu'une fonction privée doive avoir moins de responsabilités parce qu'elle est privée (j'ai ce pattern en tête, j'aimerais bien vérifier d'où je tiens ça), ceci dit trop de fonctions privées dans une fonction ça peut être le signe que la fonction fait trop de choses ^^) et ça serait plus facile à tester (on lui passe un mock)

Reply:
Comme discuté ensemble, j'ai passé l'extension en paramètre de la fonction 😄